Core/MMKV_IO.cpp

```diff
@@ -559,6 +559,16 @@ thread_local AESCryptStatus t_status;
 #    endif
 #endif // MMKV_DISABLE_CRYPT
 
+MMBuffer MMKV::readRealData(mmkv::MMBuffer & data) {
```

Comment:
This logic has little to do with the MMKV class itself. Should not be a member function of MMKV.